### ISSUE TITLE
Should I be able to use PropertyChoiceFilter for boolean property?

### ISSUE THREAD

**Issue by sealava** [ericziethen/django-property-filter-issue-reporting#6]
**Question**
Is it incorrect to use the PropertyChoiceFilter on a boolean property?

I have a model Employee which extends User. It has a property that returns user.is_active.
`class Employee(models.Model):
    """Model representing an employee - one-to-one with user"""
    user = models.OneToOneField(User, on_delete=models.CASCADE)

    #additional extended user fields will go here
    startdate = models.DateField(null=True, blank=True)
    
    @property
    def is_active(self):
        """Property to return active status of user to employee model"""
        return self.user.is_active  

    class Meta:
	    permissions = (("can_manage_emps", "ManageEmployees"),)`
I'm using PropertyChoiceFilter to try to display meaningful choices to the enduser so they can filter to only active or only inactive users.
`class EmployeeFilter(PropertyFilterSet):

    def __init__(self, *args, **kwargs):
        super().__init__(*args, **kwargs)
        
        
    ACTIVE_CHOICES = (
    (True, 'Active'),
    (False, 'Inactive'),
    )
    
    #inactive doesn't work in this one
    active = PropertyChoiceFilter(choices=ACTIVE_CHOICES,field_name='is_active',lookup_expr='exact',label='Active')
    
    #this works but options unknown/yes/no are not intuitive to users
    #isactive = PropertyBooleanFilter(field_name='is_active',lookup_expr='exact',label='Active')
    

    class Meta:
        model = Employee
        exclude = ['startdate','user']`

When using the PropertyChoiceFilter
- if user selects nothing in the filter, it correctly shows all employees
- if user selects Active in the filter, it correctly shows only active employees
- but if user selects Inactive in the filter, it shows only active employees instead of only inactive employees

I'm not sure if this is not working as expected or if I'm attempting to use it in an unintended way.

**Comment by ericziethen** [comment]
Hi @sealava 

This should work but it looks like this is a defect.
I will raise it as a bug.

In the meantime you might want to try the following using the PropertyBooleanFilter

```
from django_filters.widgets import BooleanWidget
...

ACTIVE_CHOICES = (
(True, 'Active'),
(False, 'Inactive'),
)
my_widget = BooleanWidget()
my_widget.choices = ACTIVE_CHOICES

isactive = PropertyBooleanFilter(widget=my_widget, field_name='is_active',lookup_expr='exact',label='Active')
```

**Comment by sealava** [comment]
Thank you! I appreciate your time and investigation.

**Comment by ericziethen** [comment]
This has now been fixed and is part of release 1.1 which is now available via pip.
